test/Microsoft.IdentityModel.TestUtils/TokenValidationExtensibility/CustomTokenReplayValidationDelegates.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Tokens;
+
+#nullable enable
+namespace Microsoft.IdentityModel.TestUtils
+{
+    internal class CustomTokenReplayValidationDelegates
+    {
+        internal static ValidationResult<DateTime?> CustomTokenReplayValidationDelegate(
```

Comment:
nit: maybe rename to clarify that this throws an error: `CustomTokenReplayValidationErrorDelegate`?